tmux/demos/medical/Kitware-R18.yml

```diff
@@ -57,13 +57,28 @@ windows:
       layout: even-vertical
       panes:
         # Read sensor input from bag file.
-        - sensor_input: echo ros2 bag play \
-            ${ANGEL_WORKSPACE_DIR}/ros_bags/rosbag2_2024_04_10-18_57_36-R18_Chris_live/
-            --remap /kitware/image_0:=/kitware/PVFramesBGR
+        - sensor_input: echo ros2 run angel_system_nodes hl2ss_ros_bridge --ros-args
```

Comment:
This is duplicating the `sensor_input_stream` section below. It was put there and commented out with the intention that we could swap back and forth between bags and the HL2 by commenting and uncommenting these sections.

Reply:
agreed - I swapped the 2 for now